src/app/app/messages/_components/MessageListener.tsx

```
@@ -1,45 +1,72 @@
-  }, [threadId, onNewMessage]);
+    let cancelled = false;
+
+    async function decryptAll() {
+      const results = await Promise.all(
+        messages.map(async (msg) => {
+          const content = msg.isEncrypted
+            ? await decryptMessage(threadId, msg.content)
+            : msg.content;
+          return {
+            id: msg.id,
+            content,
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'content' should be before 'id'.

(sort-keys-fix/sort-keys-fix)